### PR TITLE
pushbutton props

### PR DESCRIPTION
**Pull request by Abse2001**

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -82,6 +82,9 @@ export const ledProps = commonComponentProps.extend({
 export const ledPins = lrPolarPins
 export type LedProps = z.input<typeof ledProps>
 
+export const pushbuttonProps = commonComponentProps.extend({})
```
Comment (seveibar) [inline lib/index.ts:85]:
```suggestion
export const pushButtonProps = commonComponentProps.extend({})
```

**Review by seveibar — CHANGES_REQUESTED** [review]
Create a separate file, there is a notice at the top of lib/index.ts

**Comment by Abse2001** [comment]
Will do

File: lib/components/pushbutton.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { commonComponentProps } from "lib/common/layout"
```
Comment (seveibar) [inline lib/components/pushbutton.ts:1]: filename should match casing `push-button.ts`

**Review by seveibar — CHANGES_REQUESTED** [review]

**Review by seveibar — APPROVED** [review]

**Pull request merged**